### PR TITLE
Resprites TTVs (Sprites from DimWhat)

### PR DESCRIPTION
**Pull request by Zonespace27**
[QoL]
## About the PR 
This PR resprites TTVs. In addition, it adds some signals for bomb triggers, which are used to update the TTV sprite, since now the triggers (signaller, timer, prox sensor) have their own, unique sprites
image
(the tank2 butt sprite is slightly off but that has since been rectified)

## Why's this needed? 
TTV sprites are old, crusty, cabinet perspective, and these are miles better.

### CODE REVIEW

File: code/obj/item/device/transfer_valve.dm [hunk 1]
```diff
@@ -176,9 +187,10 @@
 				toggle = 1
 
 	process()
+		if(signalled)
+			UpdateIcon()
```
Comment (ZeWaka) [inline code/obj/item/device/transfer_valve.dm:191]: This is really weird behavior to me - why not just  have a dirty icon var and set it to false at the end of icon updating

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/item/device/transfer_valve.dm (hunk 1 above)
Comment (Zonespace27) [reply code/obj/item/device/transfer_valve.dm:191]: Would you mind elaborating a bit on the first part of that?

File: code/obj/item/device/transfer_valve.dm [hunk 2]
```diff
@@ -321,6 +353,12 @@
 		c_state()
 			return
 
+		signal_start()
+			signalled = TRUE
```
Comment (ZeWaka) [inline code/obj/item/device/transfer_valve.dm:357]: .

File: code/obj/item/device/transfer_valve.dm (hunk 1 above)
Comment (ZeWaka) [reply code/obj/item/device/transfer_valve.dm:191]: actually my idea wouldn't really work but why wouldn't you put the updateicon code below

**Review by ZeWaka — COMMENTED** [review]
^

File: code/obj/item/device/transfer_valve.dm (hunk 1 above)
Comment (ZeWaka) [reply code/obj/item/device/transfer_valve.dm:191]: that's not what i was saying - why not just literally put the `UpdateIcon()` call in signal_start

Comment (Zonespace27) [reply code/obj/item/device/transfer_valve.dm:191]: certain things (timer and prox sensor come to mind) change their sprite based on how much time is left or certain factors, so they'll need their sprites to be updated accordingly

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**